src/components/CalendarPage.js

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useState } from "react";
+import { useSelector } from "react-redux";
+import { format, add, getDay } from "date-fns";
+import CalenderBox from "./CalendarBox";
+import styled from "styled-components";
+import axios from "axios";
+
+const Container = styled.div`
+  display: flex;
+  margin-bottom: 20px;
+  justify-content: center;
+  align-items: center;
+  border-collapse: collapse;
+`;
+
+export default function CalendarPage() {
+  const [currentDate, setCurrentDate] = useState(new Date());
+  const [todos, setTodos] = useState([]);
+  const loginState = useSelector(state => state.user.loginSucceed);
+
+  const WEEK = ["일", "화", "수", "목", "금", "토", "월"];
+  const handleLeft = () => {
+    setCurrentDate(add(currentDate, { days: -7 }));
+  };
+  const handleRight = () => {
+    setCurrentDate(add(currentDate, { days: 7 }));
+  };
```

Comment:
요것도 시간적인 의미에 맞게 네이밍만 조금 고려해봤습니다. 적당히 취사 선택해서 최종 반영해주시면 될 것 같습니다.

```suggestion
  const handlePrevButtonClick = () => {
    setCurrentDate(add(currentDate, { days: -7 }));
  };
  const handleNextButtonClick = () => {
    setCurrentDate(add(currentDate, { days: 7 }));
  };
```